public/components/event_analytics/explorer/log_explorer.scss

```diff
@@ -32,4 +32,8 @@
 
 .search-area {
   position: relative;
+}
+
+.euiIcon {
```

Comment:
minor: Can you be more specific with classname. `.euiIcon` is very generic. We have had issues in the past where generic scss classnames influenced the Dashboards components in other plugins too. Something like `logs-eui-icon` can be better here.

Reply:
got it , will update it